modules/ROOT/pages/design-import-files.adoc

```diff
@@ -41,8 +41,8 @@ The fragments that you import can be written in RAML 0.8 or RAML 1.0. You can im
 * To import an API specification or API fragment from Exchange, together with its dependencies:
 . Open the *Import Asset from Exchange* dialog by using either of these methods:
 +
-** In the code editor only: If your project contains only the default file that API Designer created together with the project, click the linked text that appears in the *Files* pane.
-** In the code editor only: If your project contains at least one file that you have created, click the three-dot icon at the top of the left pane and select *Import from Exchange*.
+** *In the code editor only:* If your project contains only the default file that API Designer created together with the project, click the linked text that appears in the *Files* pane.
+** *In the code editor only:* Click *Settings* in the top-right corner of the code editor and select *Import from Exchange*.
 ** In the visual editor only: If your project does not yet contain any resources or data types, click the linked text that appears in the *New API* pane.
```

Comment:
```suggestion
** In the visual editor only
If your project does not yet contain any resources or data types, click the linked text that appears in the *New API* pane.
```
Note that this is not boldfaced like the previous bullet item is: standardize.

Reply:
Resolved in a different way.